<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_3Class</class>
 <widget class="QMainWindow" name="Test_3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test_3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>611</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>711</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../Pictures/wallpapers/branch_flowers_spring_90708_1920x1080.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niagara Solid</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FILTER </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="precipitationEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niagara Solid</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter precipitation:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niagara Solid</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RETURN </string>
    </property>
   </widget>
   <widget class="QPushButton" name="hourButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>590</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niagara Solid</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GET HOURS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="descriptionEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>550</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niagara Solid</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter description:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>609</width>
      <height>502</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niagara Solid</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DAILY WEATHER</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="test_3.qrc"/>
 </resources>
 <connections/>
</ui>
